samples/zigbee/light_switch/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8)
+
+add_definitions(-DZB_ED_ROLE)
+
+include($ENV{ZEPHYR_BASE}/../nrf/cmake/boilerplate.cmake)
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
```

Comment:
Use `find_package(Zephyr ...)`, same as in other samples (this was reworked recently).